workshops/forms.py

```diff
@@ -1036,6 +1037,32 @@ class Meta:
         }
 
 
+class TrainingRequestUpdateForm(forms.ModelForm):
+    person = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.PersonLookup,
+        label='Matched Person',
+        required=False,
+        help_text=AUTOCOMPLETE_HELP_TEXT,
+        widget=selectable.AutoComboboxSelectWidget,
+    )
+
+    class Meta:
+        model = TrainingRequest
+        exclude = ()
```

Comment:
Recently you declared to start using `fields` instead of `exclude` to avoid some mistakes :-)